pkg/transportrequest/cts/upload.go

```diff
@@ -119,7 +119,7 @@ func (action *UploadAction) Perform(command command.ShellRunner) error {
 
 	cmd = append(cmd, deployStatement)
 
-	return command.RunShell("/bin/bash", strings.Join(cmd, "\n"))
+	return command.RunShell("/bin/sh", strings.Join(cmd, "\n"))
```

Comment:
@marcusholl Could you comment if this is alright?

Reply:
Discussed that on the phone with @srinikitha09 this morning. The reason why I used `/bin/bash` is that I thought `/bin/bash` would be generally available these days ... .

Just switch to `/bin/sh` and check if it still works. I personally consider `/bin/bash` as some kind of default shell. But yes, for the alpine image(s) `/bin/bash` is stripped out.